package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "weather-app",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
```

Comment:
- [ ] Please include `    "dev": "webpack --mode development",` in your scripts.

Reply:
Done!